sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -45,6 +45,14 @@
   text-transform: uppercase;
 }
 
+.category-header-small-no-transform {
+  font-style: $font-style-normal;
+  font-weight: $formio-category-header-wt;
+  font-size: $formio-content;
+  line-height: $line-height-header-sm;
+  letter-spacing: 0.5px;
+}
```

Comment:
If getting rid of  text-transform: uppercase from category header small is the only intention here then you can use @extend from scss.

e.g 
_.category-header-small-no-transform {
@extend .category-header-small;
text-transform: none;
}_